src/main/java/codeu/model/data/Mention.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 Google Inc.
+
+package codeu.model.data;
+
+import java.time.Instant;
+import java.util.UUID;
+
+/**
+The Mention class adds the ability to mention a user. 
+It also notifies users who have been mentioned
+ */
+public class Mention {
+
+	  private final UUID idOfMentioner;// the id of this User mentioning another user
```

Comment:
Also, since UUID is just a generic id, you should put the entity type that it points to in the variable name.  So: instead of idOfMentioner, try userIdOfMentioner.  Or: mentionerUserId.